docs/quickstarts/astro.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Astro Quickstart
+description: Add authentication and user management to your Astro app with Clerk.
+---
+
+# Astro Quickstart
+
+<TutorialHero
+  framework="astro"
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "Create a Astro application",
+      link: "https://docs.astro.build/en/install-and-setup",
+      icon: "astro",
+    },
+  ]}
+  exampleRepo={[
+    {
+      title: "Astro Quickstart Repo",
+      link: "https://github.com/clerk/clerk-astro-quickstart"
+    }
+  ]}
+>
+
+- Install `@clerk/astro`
+- Set up your environment keys to test your app locally
+- Add TypeScript declarations
+- Add the `clerk()` integration to your application
+- Use Clerk middleware to implement route-specific authentication
+- Create a header with Clerk components for users to sign in and out
+
+</TutorialHero>
+
+<Steps>
+### Install `@clerk/astro`
+
+Clerk's [Astro SDK](/docs/references/astro/overview) has prebuilt components and helpers to make user authentication easier.
+
+To get started using Clerk with Astro, add the SDK to your project:
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+  ```bash filename="terminal"
+  npm install @clerk/astro
+  ```
+
+  ```bash filename="terminal"
+  yarn add @clerk/astro
+  ```
+
+  ```bash filename="terminal"
+  pnpm add @clerk/astro
+  ```
+</CodeBlockTabs>
+
+### Set environment variables
+
+<SignedIn>
+
+Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
```

Comment:
```suggestion
Add the following keys to your `.env.local` file. These keys can be found in the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
```

Reply:
What we originally added was as text followed the pattern we have for Expo and Nextjs. So it's not astro specific. You may wanna open another PR in the future to tackle this. 
Probably i'm not the right person to decide if we want this change or not, maybe @royanger or @alexisintech can.